### PR TITLE
Create model.rst

### PR DESCRIPTION
**Pull request by srcodesmith**
Correction of awkward phrasing.

Each pull request should address a single issue and have a meaningful title.

**Description**
Don't want to inadvertently step on anyone's toes, but, being new to CI, I am going through the mostly extremely well-written docs, but encountering the odd place where it seems to have been written by someone whose first language was NOT English. Might I help out to make corrections in such cases? I hope I'm not being presumptuous in choosing this route to do so.
I will stop here and await responses. 

**Checklist:**
- [ ] Securely signed commits
- [ ] Component(s) with PHPdocs
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/models/model.rst [hunk 1]
```diff
@@ -187,7 +187,7 @@ data type.
 
 **$createdField**
 
-Specifies which database field should use for keep data record create timestamp.
+Specifies which database field to use for data record create timestamp.
```
Comment (paulbalandan) [inline user_guide_src/source/models/model.rst:190]: "data record create timestamp" seems off to me.

File: user_guide_src/source/models/model.rst [hunk 2]
```diff
@@ -187,7 +187,7 @@ data type.
...
 Leave it empty to avoid update it (even useTimestamps is enabled)
```
Comment (paulbalandan) [inline user_guide_src/source/models/model.rst:191]:
```suggestion
Leave it empty to avoid updating it (even if ``$useTimestamps`` is enabled)
```

**Review by paulbalandan — COMMENTED** [review]
I doesn't have the chance to browse over the documentation's grammar nitty-gritties. A little helping hand to preen these grammar errors would be very much appreciated.

File: user_guide_src/source/models/model.rst (hunk 1 above)
Comment (srcodesmith) [reply user_guide_src/source/models/model.rst:190]: Yes, I agree with you. Just didn't want to go farther before getting positive feedback.

File: user_guide_src/source/models/model.rst (hunk 2 above)
Comment (srcodesmith) [reply user_guide_src/source/models/model.rst:191]: Yes, very good, see my previous comment above.  If you would like me to continue with these "nitty-gritties" I'd be delighted to do so as I peruse the docs and notice these anomalies.  I'll try to get better at the commits, and try to be better about the descriptions. Please give me feedback as required.
Again, aside from these little hiccups, I'm very impressed by the docs!

- [ ]

**Comment by paulbalandan** [comment]
@srcodesmith Yes, please, go on. Every helping hand from the community is very much needed and welcome.

File: user_guide_src/source/models/model.rst (hunk 1 above)
Comment (paulbalandan) [reply user_guide_src/source/models/model.rst:190]: Ok then. Can you please go further and edit "data record create timestamp" to make it less confusing?

File: user_guide_src/source/models/model.rst (hunk 2 above)
Comment (paulbalandan) [reply user_guide_src/source/models/model.rst:191]: Please do so. Your help is appreciated.

**Comment by paulbalandan** [comment]
@srcodesmith can you do my review suggestions so that I can merge this?

**Comment by srcodesmith** [comment]
Hi Paul,
I THINK i've done what's necessary.
Sorry I haven't made more suggestions yet, but I'd like to uncover several
glitches and submit them as a group, if that makes sense.

**Comment by srcodesmith** [comment]
Pretty much identical working needs to be changed for the entry immediately below: $updatedField, as well.

**Review by paulbalandan — APPROVED** [review]
You could create those edits in a new PR I guess. Merging this.

**Pull request merged**